package.json

```diff
@@ -33,16 +33,16 @@
   },
   "homepage": "https://github.com/google/model-viewer#readme",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.3.0",
-    "@typescript-eslint/parser": "^4.3.0",
-    "clang-format": "^1.4.0",
-    "eslint": "^7.7.0",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
+    "@typescript-eslint/parser": "^4.28.3",
+    "clang-format": "^1.5.0",
+    "eslint": "^7.30.0",
     "eslint-config-google": "^0.14.0",
-    "eslint-plugin-mocha": "^8.0.0",
-    "eslint-plugin-wc": "^1.2.0",
-    "http-server": "^0.12.1",
-    "husky": "^4.2.1",
-    "lerna": "^3.22.1",
-    "typescript": "4.0.3"
+    "eslint-plugin-mocha": "^9.0.0",
+    "eslint-plugin-wc": "^1.3.0",
+    "http-server": "^0.12.3",
+    "husky": "^7.0.1",
+    "lerna": "^4.0.0",
+    "typescript": "4.3.5"
```

Comment:
FYI, when this gets merged you'll all want to do an `npm run clean` and `npm bootstrap` before building. Let me know if you run into any issues.